fixed_bed_model.py

```diff
@@ -0,0 +1,2630 @@
+        if(len(T) > 1):
+            h = plt.contourf(T,Z,den1)
+            plt.xlabel('Time [sec]')
+            plt.ylabel('Bed Position [scaled]')
+            plt.colorbar(h)
+            plt.title(c +' Density [mol/m$^3$]')
+        else:
+            plt.plot(Z,den1)
+            plt.xlabel('Bed Position [scaled]')
+            plt.ylabel(c +' Density [mol/m$^3$]')
+        plt.show()
+
+    if m.chemsorb:
+        
+        # Visualize isotherm
+        nchemstar1, nchemstar2 = extract3d(m, m.nchemstar,'CO2')
+        if(len(T) > 1):
+            h = plt.contourf(T,Z,nchemstar1)
+            plt.xlabel('Time [sec]')
+            plt.ylabel('Bed Position [scaled]')
+            plt.colorbar(h)
+            plt.title('Chemical Isotherm [mol/kg]')
+        else:
+            plt.plot(Z,nchemstar1)
+            plt.xlabel('Bed Position [scaled]')
+            plt.ylabel('Chemical Isotherm [mol/kg]')
+        plt.show()
+        
+        # Visualize loading
+        nchem1, nchem2 = extract3d(m, m.nchem,'CO2')
+        if(len(T) > 1):
+            h = plt.contourf(T,Z,nchem1)
+            plt.xlabel('Time [sec]')
+            plt.ylabel('Bed Position [scaled]')
+            plt.colorbar(h)
+            plt.title('Chemical Loading [mol/kg]')
+        else:
+            plt.plot(Z,nchem2)
+            plt.xlabel('Bed Position [scaled]')
+            plt.ylabel('Chemical Loading [mol/kg]')
+        plt.show()
+        
+    if m.physsorb:
+        nphysstar1, nphysstar2 = extract3d(m, m.nphysstar,'CO2')
+        if(len(T) > 1):
+            h = plt.contourf(T,Z,nphysstar1)
+            plt.xlabel('Time [sec]')
+            plt.ylabel('Bed Position [scaled]')
+            plt.colorbar(h)
+            plt.title('Physical Isotherm [mol/kg]')
+        else:
+            plt.plot(Z,nphysstar1)
+            plt.xlabel('Bed Position [scaled]')
+            plt.ylabel('Physical Isotherm [mol/kg]')
+        plt.show()
+           
+        nphys1, nphys2 = extract3d(m, m.nphys,'CO2')
+        if(len(T) > 1):
+            h = plt.contourf(T,Z,nphys1)
+            plt.xlabel('Time [sec]')
+            plt.ylabel('Bed Position [scaled]')
+            plt.colorbar(h)
+            plt.title('Physical Loading [mol/kg]')
+        else:
+            plt.plot(Z,nphys1)
+            plt.xlabel('Bed Position [scaled]')
+            plt.ylabel('Physical Loading [mol/kg]')
+        plt.show()
+        
+        alpha1, alpha2 = extract3d(m, m.alpha,'CO2')
+        if(len(T) > 1):
+            h = plt.contourf(T,Z,alpha1)
+            plt.xlabel('Time [sec]')
+            plt.ylabel('Bed Position [scaled]')
+            plt.colorbar(h)
+            plt.title('alpha is modified isotherm [dimensionless]')
+        else:
+            plt.plot(Z,alpha1)
+            plt.xlabel('Bed Position [scaled]')
+            plt.ylabel('alpha is modified isotherm [dimensionless]')
+        plt.show()
+
+
+### Functions that are no longer used
+def init1D(dyn_var, t0_var,set1,T):
```

Comment:
Remove functions that are no longer used. They are stored in the GitHub history.